tools/util_windows_test.go

```diff
@@ -0,0 +1,111 @@
+// +build windows
+
+package tools
+
+import (
+	"crypto/sha1"
```

Comment:
If we're going to use a hash for comparing two files, let's use SHA-256.  SHA-1 is considered weak and we'd like to use less of it in our codebase, and we already use SHA-256 in a lot of places.

Reply:
Got it. Thank you.